### PR TITLE
Fix network path

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-modules/issues/531

### CODE REVIEW

**Comment by shargon** [comment]
@cloud8little clould you check it now?

**Review by erikzhang — DISMISSED** [review]

**Comment by cloud8little** [comment]
cbcf3bf69d0506e4cd880371fab3992

File: src/RpcNep17Tracker/RpcNep17Tracker.cs [hunk 1]
```diff
@@ -38,16 +39,14 @@ protected override void OnSystemLoaded(NeoSystem system)
         {
             if (system.Settings.Magic != _network) return;
             System = system;
+            string path = string.Format(_dbPath, system.Settings.Magic.ToString("x8"));
```
Comment (cloud8little) [inline src/RpcNep17Tracker/RpcNep17Tracker.cs:42]:
```suggestion
            string path = string.Format(_dbPath, system.Settings.Magic.ToString("X8"));
```

**Review by erikzhang — APPROVED** [review]

**Comment by cloud8little** [comment]
Test passed:

1. "DBPath" in config file has "{0}" string, and network id equal to magic id. Folder (Nep17BalanceData_004F454E) generate successfully.  
2.  "DBPath" in config file has not symbol similar with "{0}" and network id equal to magic id, folder with the string in config file generate successfully.
3.  network id not same with magic id, there exist a folder name with the config file in "DBPath" field, no data generated.
if there is no folder name with the "DBPath" field, no folder generated.

**Review by cloud8little — APPROVED** [review]

**Pull request merged**